### PR TITLE
Add configs

### PR DESCRIPTION
**Pull request by mattfullerton**
Ability to use local file storage instead of s3
Ability to serve to only certain IPs (useful when using a test instance for embargoed data)

### CODE REVIEW

**Comment by pudo** [comment]
I obviously love the file support, but the limit IP thing seems to me like something that should be done in a higher-level web server (nginx, apache). Are you sure we need it in core?

**Comment by mattfullerton** [comment]
We absolutely do not :) But when quickly developing something on a home PC I have neither nginx, apache or AWS security groups (and this could be quite a common developer use case... embargoed data etc). Would you like me to prune down the PR?

**Comment by pudo** [comment]
wdyt, @pwalsh @milafrerichs ?

**Comment by pwalsh** [comment]
File support is excellent. Very much looking forward to getting this into OS.

As for the IP limit thing, I think it is fine here, and useful as @mattfullerton points out. I've personally done this type of thing on various django and flask code bases.

**Comment by breyten** [comment]
+1 for file support. As for the IP settings maybe something like this:

```
allowed_ips = app.config.get('ALLOWED_IPS', [])
if len(allowed_ips) > 0 and request.remote_addr not in allowed_ips:
    abort(403)
```

This way you have one setting, instead of two. (But this is a bit of nitpicking ;))

**Comment by milafrerichs** [comment]
File support is excellent. But I would rather see the settings made available via the environment.

e.g.:
```python
STORAGE_TYPE = env('SPENDB_STORAGE_TYPE', 's3')
STORAGE_PATH = env('SPENDB_STORAGE_PATH', 'default/path')
```

not sure about the IP limiting...

**Comment by pudo** [comment]
Hm, so I really do feel like this would be better to keep apart and merge the storage, while keeping the IP filtering to front-end tools?

**Comment by mattfullerton** [comment]
OK, I've removed IP support and added defaults/env variable loading as suggested by @milafrerichs

File: prod_settings.py [hunk 1]
```diff
@@ -19,6 +19,12 @@
 
 MAIL_DEFAULT_SENDER = env('SPENDB_MAIL_SENDER', 'SpenDB <info@mapthemoney.org>')
 
+STORAGE_TYPE = 's3' #Alternative: 'file'
+STORAGE_PATH = '/a/path/only/valid/if/using/local/file/system'
+
+STORAGE_TYPE = env('SPENDB_STORAGE_TYPE', 's3') #Alternative: 'file'
+STORAGE_PATH = env('SPENDB_STORAGE_PATH', '/usr/local/lib/spendb') #Only used if 'file' selected above
```
Comment (milafrerichs) [inline prod_settings.py]: no need to define it twice. can you remove the lines 22 & 23?

**Comment by milafrerichs** [comment]
see the comment, otherwise looks great. thanks @mattfullerton

**Comment by mattfullerton** [comment]
@milafrerichs that's what happens to commits at 7:23... removed the duplication

**Comment by pudo** [comment]
You guys rock :)

**Pull request merged**